packages/office-ui-fabric-react/src/demo/pages/ScreenReaderAlertPage/ScreenReaderAlertPage.tsx

```diff
@@ -0,0 +1,98 @@
+const ScreenReaderAlertSpinnerExampleCode = require('./examples/ScreenReaderAlert.Spinner.Example.tsx');
+
+export class ScreenReaderAlertPage extends React.Component<IComponentDemoPageProps, any> {
+  private _url: string;
+
+  constructor() {
+    super();
+    this._url = getPageRouteFromState(AppState, 'UTILITIES', 'ScreenReaderAlert');
+  }
+
+  public render() {
+    return (
+      <ComponentPage
+        title='ScreenReaderAlert'
+        componentName='ScreenReaderAlertExample'
+        exampleCards={
+          <div>
+            <ExampleCard title='ScreenReaderAlert basic' code={ ScreenReaderAlertBasicExampleCode }>
+              <ScreenReaderAlertBasicExample />
+            </ExampleCard>
+            <ExampleCard title='ScreenReaderAlert repeat same message' code={ ScreenReaderAlertRepeatExampleCode }>
+              <ScreenReaderAlertRepeatExample />
+            </ExampleCard>
+            <ExampleCard title='ScreenReaderAlert with Spinner' code={ ScreenReaderAlertSpinnerExampleCode }>
+              <ScreenReaderAlertSpinnerExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
+          <PropertiesTableSet componentName='ScreenReaderAlert' />
+        }
+        overview={
+          <div>
+            <p>
+              ScreenReaderAlerts are used to read out the notifications on the page to the blind people with the help of screen readers like JAWS, Chrome VOX and NVDA. It is implemented by live region and overcomes some known issues about live region when using with React.
+            </p>
+            <p>
+              This component has been tested through following combination of screen readers and browsers:
+            </p>
+            <ul>
+              <li>
+                <p>1. <strong>JAWS 18</strong> with <strong>IE 11</strong>.</p>
+              </li>
+              <li>
+                <p>2. <strong>Chrome VOX v53</strong> with <strong>Chrome v54</strong>.</p>
+                <p>    Note: ReadingMode.ReadAfterOtherContent option is currently not supported in ChromeVOX, text will be read immediately.</p>
+              </li>
+            </ul>
+
+            <p>
+              To try the demo in this page, you should at least using one kind of screen reader. E.g. JAWS, ChromeVOX, NVDA.
+              Better use the browser recommendated by the screen reader you're using. E.g. IE11 with JAWS, Chrome with ChromeVOX, FireFox with NVDA.
+            </p>
+          </div>
+        }
+        bestPractices={
+          <div></div>
+        }
+        dos={
+          <div>
+            <ul>
+              <li>Use if it would be hard for blind people to know dynamic changes on the page.</li>
+              <li>Pay attention to user experience for blind users: do they have enough information they need?</li>
+              <li>Use it to notify user the status change of async tasks.</li>
```

Comment:
Please call out that where there is visible text/ui, those should be used as the true live region. This should really just be used as a supplement.